rancher/dashboard
# Move locale user preferences

### Summary
Fixes #6660

### Occurred changes and/or fixed issues
- Moved local from Side nav to preference page 
- Local style should change as per the page location

### How to test

- The login page should have a Local selector
- After logging in side nav should not have a Local selector, it should be on the user preference page
- Style of the Local selector should change as per the page.
